tilelang/contrib/cutedsl/ptx_mma.py

```
@@ -0,0 +1,857 @@
+# =============================================================================
+
+
+@dsl_user_op
+def ptx_mma_m16n8k16_f16_f16_f32(
+    a_ptr: Pointer,
+    a_offset,
+    b_ptr: Pointer,
+    b_offset,
+    c_ptr: Pointer,
+    c_offset,
+    a_layout: str = "row",
```

Ruff (0.15.1)

[warning] Unused function argument: `a_layout`

(ARG001)